src/LinAlg/hiopVectorRajaPar.cpp

```diff
@@ -413,7 +437,7 @@ void hiopVectorRajaPar::copyToStarting(int start_index, hiopVector& dst)
  * @pre start_index + vec.n_local_ <= n_local_
  * @pre `this` and `vec` are not distributed
  */
-void hiopVectorRajaPar::copyToStarting(hiopVector& vec, int start_index/*_in_dest*/)
+void hiopVectorRajaPar::copyToStarting(hiopVector& vec, int start_index/*_in_dest*/) const
 {
```

Comment:
```suggestion
void hiopVectorRajaPar::copyToStarting(hiopVector& vec, int start_index_in_dest) const
{
```
Why not just expand the variable name instead of using a comment?